packages/helpermodules/modbusserver.py

```diff
@@ -52,24 +52,27 @@ def _form_int16(value, startreg):
 
 
 def _form_str(value: str, startreg):
```

Comment:
Müsste hier dann nicht mit `Optional` gearbeitet werden, damit `None` auch ein zulässiger Typ ist?
```suggestion
def _form_str(value: Optional[str], startreg):
```